workspaces/adventure-pack/goodies/typescript/Iterator.prototype.tally/index.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "../Iterator.prototype.filter";
+import "../Iterator.prototype.map";
+
+import { iteratorPrototype } from "../Iterator.prototype";
+delete (iteratorPrototype as unknown as Record<string, unknown>).toArray;
```

Comment:
We have to delete things like `.map` and `.filter` to make sure we're testing our own versions, but I think `.toArray` doesn't currently exist and isn't planned, so we probably don't have to worry about deleting it.